libraries/SITL/SIM_EFI_EMUECU.cpp

```diff
@@ -0,0 +1,102 @@
+/*
+   This program is free software: you can redistribute it and/or modify
+   it under the terms of the GNU General Public License as published by
+   the Free Software Foundation, either version 3 of the License, or
+   (at your option) any later version.
+
+   This program is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+   GNU General Public License for more details.
+
+   You should have received a copy of the GNU General Public License
+   along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+/*
+  simulate MegaSquirt EFI system
+*/
+
+#include "SIM_Aircraft.h"
+#include <SITL/SITL.h>
+#include <AP_HAL/utility/sparse-endian.h>
+#include <stdio.h>
+
+using namespace SITL;
+
+// printf to CAN LogMessage for debugging
```

Comment:
Nope, definitely not going to be doing that.

Reply:
fixed